python/ray/data/context.py

```diff
@@ -239,6 +239,10 @@ def __init__(
         # The max number of blocks that can be buffered at the streaming generator of
         # each `DataOpTask`.
         self._max_num_blocks_in_streaming_gen_buffer = None
+        # Whether to enable ReservationOpResourceLimiter.
+        self.op_resource_reservation_enabled = False
```

Comment:
Are we planning on adding an environment variable for this? More generally, when don't we want to add an environment variable for a setting? Seems like most do right now.

Reply:
We should add environment varilable and add into parameter as other configs.

Reply:
will add an env var for this. 
but regarding parameter of `__init__`, I don't think it's useful anymore. because there are too many parameters already. and now we already don't construct a DataContext with `__init__` params.

Reply:
> but regarding parameter of __init__, I don't think it's useful anymore. because there are too many parameters already. and now we already don't construct a DataContext with __init__ params.

We can clean it up later. @raulchen - is that you cannot add to `__init__` because smoething is failing?

Reply:
oh no, nothing was failing. I just think we should stop adding new params, and clean up the existing ones later.